packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -526,6 +526,9 @@ class Overlay extends StatefulWidget {
 class OverlayState extends State<Overlay> with TickerProviderStateMixin {
   final List<OverlayEntry> _entries = <OverlayEntry>[];
 
+  /// all OverlayEntry
+  List<OverlayEntry> get entries => _entries;
```

Comment:
Exposing this will allow developers to directly modify this list, which is guaranteed to lead to bugs as the Overlay will not properly update. Therefore, we should not expose this.